app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express")
+const { error500 } = require('./controllers/error-handling-controller.js')
+const app = express()
+app.use(express.json())
```

Comment:
We shouldn't have this express.json line until we actually need it

(we use it when we're got request.bodies with JSON that needs to be parsed - not yet!)